package.json

```diff
@@ -22,10 +22,6 @@
   ],
   "author": "joshwnj",
   "license": "MIT",
-  "peerDependencies": {
```

Comment:
I need to learn more about `peerDependencies` :) Can you elaborate on the reason for moving it? 

Perhaps @kof could chime in too, as he initiated it here: https://github.com/joshwnj/react-visibility-sensor/commit/a68451e58631416243368bbd56b05e25f661845d